web/templates/user/signup3.html.eex

```diff
@@ -1,5 +1,8 @@
 <div class="ma3 ma4-m ma4-l mv2 mv3-m mv4-l">
   <h3 class="tc">Sign up</h3>
+
+  <p>By Signing to Healthlocker, I agree that...</p>
```

Comment:
where was this copy requested? (_please link to the issue in your commit message..._)